Header header
float32 drone_timeuint16 tag
uint16 size
float32 vision_tx_raw
float32 vision_ty_raw
float32 vision_tz_raw
